src/config/models.py

```diff
@@ -52,6 +52,11 @@ class ComparisonType(enum.StrEnum):
     smaller = enum.auto()
 
 
+class ForceRefreshType(enum.StrEnum):
```

Comment:
Do you think we add new items to this enum? If not, remove it and change the arg inside the function to bool

Reply:
There are 3 options here: Do no force refresh (I use None for that), force refresh with no filter, force refresh with a filter.

Would it make more sense then for me to add that third option to the enum instead of using None?

Reply:
Just to add - I doubt new options will ever be added.

Reply:
Yes instead of magical value None, lets just add the third option here and replace the None checks